### DISCUSSION TITLE
Action request: Add related item

### DISCUSSION THREAD

**Discussion by ulahcherubim** [windingwind/zotero-actions-tags#247] [Ideas]
I need to open selection for adding related item via keyboard shortcut. I know there is an action already for relating two selected items, however I need to relate notes to notes and items while typing my notes.

**Comment by nikbpetrov** [comment]
My super silly workaround for Zotero 7, that seems to get the job done

```js
const window = require("window");
window.document.querySelector("#zotero-editpane-related toolbarbutton.add image").click()
```

It opens 2 related boxes, for some strange reason, but just press escape once right after.

And for Zotero 6: https://github.com/windingwind/zotero-actions-tags/discussions/172

**Reply by ulahcherubim** [reply]
Thanks for this. It's interesting that it opens two tabs. Aside from that, does the job!

**Reply by ramonmi** [reply]
Modified to prevent duplication of related dialogs.
```js
/**
 * Add related item
 * @usage shortcut
 * @link https://github.com/windingwind/zotero-actions-tags/discussions/247
 * @see https://github.com/windingwind/zotero-actions-tags/discussions/247
 */
if (item) {
    return;
}
const window = require("window");
window.document.querySelector("#zotero-editpane-related toolbarbutton.add image").click()
```
